driver_cpl/cime_config/testdefs/testlist_drv.xml

```diff
@@ -672,4 +654,195 @@
       </machine>
     </machines>
   </test>
+=======
```

Comment:
Thanks for catching this. The merge added the old schema to the new. Its now fixed.